scenarios/detection/12_hard_negative_sampling.ipynb

```diff
@@ -548,7 +548,7 @@
    ],
    "source": [
     "# Get validation accuracy on test set at IOU=0.5:0.95\n",
-    "acc = float(detector.ap[-1])\n",
```

Comment:
This edit also needs to be done in the 11_ notebook since otherwise the 11_ notebook will start failing (at least) on this line:
     acc = float(detector.ap[-1])

This problem actually comes from the fact that our AML notebook testing devops pipeline is currently broken.

Reply:
Thanks.  Done.  Things getting tedious when code modification involves other collaborators' work :man_facepalming:

Reply:
@simonzhaoms Yeah, fully understand, I would have the same problem. The main cause of this problem is that the PR is so big. Good advice / painful learning to always try to add small bits of code which solve sub-problems rather than one big PR which solves the full problem.